content/about/use-case/_index.en.md

```diff
@@ -1,28 +1,99 @@
 ---
 title: Use cases
 linkTitle: Use cases
-description: Use cases and strategic axes
+description: Use cases related to rail transport planning
 weight: 60
 ---
 
-The process of planning the circulation of a train requires **10 years of anticipation** and involves the collaboration of different services and actors to transform the mobility needs of customers: The process of planning a train requires **10 years of anticipation** and involves the collaboration of different departments and actors to transform the mobility needs of customers: **organizing authorities (OA)** and **railway companies (RC)** into **annual service (AS)**.
+![OSRD presentation](osrd_presentation.png)
 
-The OSRD project aims to meet the needs of simulations that cover the entire process, from the design of the transportation plan to its operation.
+OSRD is meant to fulfill a range of use-cases related to railway planning:
+ - [timetabling](#timetabling)
+ - [operation studies](#%C3%A9tudes-dexploitation)
+ - [infrastructure management](#infrastructure-management)
+ - [short term capacity management](#short-term-capacity-management)
 
-![OSRD presentation](osrd_presentation.png)
 
-Our mission is to provide a simulation tool that can be used in **all stages of the design phase** (timetable planning and operational studies), on a **national geographic scale** (including the whole French network) and with a **variable detail size**. Moreover, in the operational phase, the OSRD project proposes a service to draw **last minute paths** (time-slot during which the infrastructure is allocated to the circulation of a specific train) in the theoretical or real residual capacity of the railway network, once the other circulations have already been planned.
+![produits OSRD](osrd_product.en.png)
+
+
+## Timetabling
+
+Timetabling is about making an transport offer in response to a
+statement of requirements.
+
+OSRD addresses this need by providing:
+ - a microscopic railway infrastructure database
+ - a rolling stock database
+ - an infrastructure edition tool
+ - a timetabling tool
+ - train path allowances, including an allowance distribution which minimizes energy use
+
+Planned features include:
+ - automated conflict detection[^conflict]
+ - macroscopic infrastructure synthesis from microscopic infrastructure
+ - interactive timetable diagrams
+ - a throughput and rolling stock turnover optimizer
+
+[^conflict]: A conflicts occurs when the movement of a train interferes with another
+
+
+## Operation studies
+
+Operation studies assess the ability of infrastructure to meet demand,
+and balance infrastructure capacity with transportation needs.
+They enable informed public infrastructure investment decisions.
+
+OSRD addresses this need by providing:
+ - a microscopic railway infrastructure database
```

Comment:
If we decide to change the corresponding bulletpoint above don't forget to change it here as well.